docs/Installation_Alternative.rst

```diff
@@ -36,13 +36,13 @@ you can use one of the following simple commands:
 
 -  using ``easy_install`` (old tool, but still works)::
 
-       easy_install --prefix /tmp easybuild
+       easy_install --prefix /tmp/$USER easybuild
```

Comment:
That's also fine by me.

I think that if somybody just copy&pastes these commands, they should end up with something that will keep on working.